.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1 @@
 ### Description
-
```

Comment:
Is this aim to just remove this for the `develop` branch?

Reply:
Nah, the PR template is just too annoying

Read the contributing section on gitflow for prod/develop 😛

Reply:
We can simplify the PR template, but it won't be removed. It's necessary that there's a standard PR description because it's become an issue in the past.

Yeah, that makes sense.